npm/install.js

```diff
@@ -71,15 +71,13 @@ function isInstalled () {
 // unzips and makes path.txt point at the correct executable
 function extractFile (zipPath) {
   return new Promise((resolve, reject) => {
-    extract(zipPath, { dir: path.join(__dirname, 'dist') }, err => {
-      if (err) return reject(err);
-
+    extract(zipPath, { dir: path.join(__dirname, 'dist') }).then(() => {
       fs.writeFile(path.join(__dirname, 'path.txt'), platformPath, err => {
         if (err) return reject(err);
 
         resolve();
       });
-    });
+    }).catch(reject);
```

Comment:
```js
return extract(zipPath, { dir: path.join(__dirname, 'dist') })
  .then(() =>
    new Promise((resolve, reject) => {
      fs.writeFile(path.join(__dirname, 'path.txt'), platformPath, err => {
        if (err) return reject(err);

        resolve();
      });
    })
  )
```

Reply:
Actually even better, if we are on node 10 and higher.  Just use `fs.promises.writeFile`

Reply:
```js
return extract(zipPath, { dir: path.join(__dirname, 'dist') })
  .then(() => fs.promises.writeFile(path.join(__dirname, 'path.txt'), platformPath))
```